### DISCUSSION TITLE
How does the pool prioritize what chain to mine?

### DISCUSSION THREAD

**Discussion by shitbox120** [dreams-money/merged-mining-pool#12] [General]
I was wondering if there is a explanation as to how the pool determines what chain to prioritize and if it makes any difference.

**Comment by dreams-money** [comment]
It doesn't make a difference.  Litecoin will always be the parent chain.  Everything else is Aux.

**Comment by shitbox120** [comment]
Can you mine DOGE by itself? or does the pool require both?

**Comment by dreams-money** [comment]
It can mine only one.  Remove one of the coins from merged_blockchain_order
